exports.js

```diff
@@ -113,6 +113,7 @@ module.exports = {
         'accessKeysLastUsed'            : require(__dirname + '/plugins/aws/iam/accessKeysLastUsed.js'),
         'accessKeysRotated'             : require(__dirname + '/plugins/aws/iam/accessKeysRotated.js'),
         'certificateExpiry'             : require(__dirname + '/plugins/aws/iam/certificateExpiry.js'),
+        'crossAccountAccess'            : require(__dirname + '/plugins/aws/iam/crossAccountAccess.js'),
```

Comment:
```suggestion
        'crossAccountMfaExtIdAccess'            : require(__dirname + '/plugins/aws/iam/crossAccountMfaExtIdAccess.js'),
```

Also, please update the plugin file name